x-pack/plugins/security_solution/public/cases/components/user_action_tree/helpers.tsx

```diff
@@ -7,22 +7,38 @@
 import { EuiFlexGroup, EuiFlexItem, EuiLink, EuiCommentProps } from '@elastic/eui';
 import React from 'react';
 
-import { CaseFullExternalService, ActionConnector } from '../../../../../case/common/api';
+import {
+  CaseFullExternalService,
+  ActionConnector,
+  CaseStatus,
+} from '../../../../../case/common/api';
 import { CaseUserActions } from '../../containers/types';
 import { CaseServices } from '../../containers/use_get_case_user_actions';
 import { parseString } from '../../containers/utils';
 import { Tags } from '../tag_list/tags';
-import * as i18n from '../case_view/translations';
 import { UserActionUsernameWithAvatar } from './user_action_username_with_avatar';
 import { UserActionTimestamp } from './user_action_timestamp';
 import { UserActionCopyLink } from './user_action_copy_link';
 import { UserActionMoveToReference } from './user_action_move_to_reference';
+import { Status, statuses } from '../status';
+import * as i18n from '../case_view/translations';
 
 interface LabelTitle {
   action: CaseUserActions;
   field: string;
 }
 
+const getStatusTitle = (status: CaseStatus) => {
```

Comment:
maybe we could make a component from that function?

Reply:
Because If I create a component from that I should do it for every title I think is best to do it as a tech dept on another PR.